index.bs

```diff
@@ -765,6 +765,8 @@ There are several subtypes of {{XRStationaryReferenceSpace}}, determined by the
 
 Note: The {{position-disabled}} subtype is primarily intended for use with pre-rendered media such as panoramic photos or videos. It should not be used for most other media types due to user discomfort associated with the lack of a neck model or full positional tracking.
 
+Devices which support {{XRReferenceSpaceType/stationary}} reference spaces MUST support all {{XRStationaryReferenceSpace/subtype}}s.
```

Comment:
I think you want to replace
  `{{XRStationaryReferenceSpace/subtype}}s`
with
  `{{XRStationaryReferenceSpaceSubtype}} values`